### PR TITLE
move Sirupsen to sirupsen on a case sensitive system

### PR DESCRIPTION
**Pull request by liangchenye**
Signed-off-by: liangchenye <liangchenye@huawei.com>

### CODE REVIEW

**Comment by liangchenye** [comment]
fix https://github.com/opencontainers/runtime-tools/issues/434 found by @mikebrow

**Comment by q384566678** [comment]
LGTM

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -7,9 +7,8 @@
 	],
 	"Deps": [
 		{
-			"ImportPath": "github.com/Sirupsen/logrus",
-			"Comment": "v1.0.0",
-			"Rev": "202f25545ea4cf9b191ff7f846df5d87c9382c2b"
+			"ImportPath": "github.com/sirupsen/logrus",
```
Comment (Mashimiao) [inline Godeps/Godeps.json]: We lost version info here. maybe you should `git checkout 1.0.2` in logrus gopath directory and then update it in runtime-tools.

**Comment by liangchenye** [comment]
updated by adding 'godeps comment'.  @Mashimiao  @q384566678

**Comment by q384566678** [comment]
need rebase.

File: Godeps/Godeps.json (hunk 1 above)
Comment (liangchenye) [inline Godeps/Godeps.json]: done

**Comment by Mashimiao** [comment]
LGTM

**Comment by q384566678** [comment]
LGTM

**Pull request merged**